app/Exceptions/UnreleasedHeadingDoesNotExistException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Exceptions;
+
+use Exception;
+
+class UnreleasedHeadingDoesNotExistException extends Exception
+{
+    public function __construct()
+    {
+        parent::__construct("The 'Unreleased' heading was not found in the CHANGELOG. Please refer to the correct CHANGELOG format at: https://github.com/bigbinary/neeto-editor/blob/master/CHANGELOG.md.");
+    }
```

Comment:
The URL shall be updated with the neetoKB article in neeto-engineering which contains a detailed explanation of how to setup the CHANGELOG file.